lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/CheckPassportHandler.java

```diff
@@ -155,6 +161,9 @@ public APIGatewayProxyResponseEvent handleRequest(
                             userId,
                             authenticationRequest.getClientID().getValue());
             passportService.persistDcsResponse(passportCheckDao);
+
+            persistResourceIdWithSessionId(passportSessionId, passportCheckDao);
```

Comment:
This lambda will now need the session DynamoDB table name env var, and Dynamo read/write permissions to the table

Reply:
Addressed in new PR.